### ISSUE TITLE
dbt run fails

### ISSUE THREAD

**Issue by dselman** [fivetran/dbt_intercom#7]
Struggling to get this working. I _think_ we have a pretty vanilla Intercom setup and we created our connector in Dec.

Here is the output from running `dbt run` locally. We use FiveTran connected to RedShift.

I've tried playing around with disabling some of the vars, but no luck.

```
Completed with 3 errors and 0 warnings:

Database Error in model stg_intercom__contact_history_tmp (models/tmp/stg_intercom__contact_history_tmp.sql)
  relation "stg_intercom__contact_history_tmp__dbt_backup" already exists
  compiled SQL at target/run/intercom_source/models/tmp/stg_intercom__contact_history_tmp.sql

Database Error in model stg_intercom__conversation_tag_history_tmp (models/tmp/stg_intercom__conversation_tag_history_tmp.sql)
  relation "intercom.conversation_tag_history" does not exist
  compiled SQL at target/run/intercom_source/models/tmp/stg_intercom__conversation_tag_history_tmp.sql

Database Error in model stg_intercom__company_tag_history_tmp (models/tmp/stg_intercom__company_tag_history_tmp.sql)
  relation "intercom.company_tag_history" does not exist
  compiled SQL at target/run/intercom_source/models/tmp/stg_intercom__company_tag_history_tmp.sql
```

**Comment by fivetran-joemarkiewicz** [comment]
Hi @dselman!

Thank you for opening this issue. Looking through your issue description I had a few quick questions:
- Do you have `conversation_tag_history` and `company_tag_history` tables in your Intercom Schema?
- Would you be able to share the variables you are declaring to hopefully help with any confusion on how to successfully deactivate those models if you do not use them.
- The first error indicates you already have a table named `stg_intercom__contact_history_tmp` in your schema. Is that true?

**Comment by dselman** [comment]
`conversation_tag_history` and `company_tag_history ` are both missing from RedShift.

![image](https://user-images.githubusercontent.com/623108/107514782-08b6be00-6ba2-11eb-84b8-b3db6f72695e.png)

**Comment by fivetran-joemarkiewicz** [comment]
Thanks for sharing @dselman! Since you do not have the conversation or company tag tables, you can disable them from the dbt project. You can add the below variable config to your `dbt_project.yml` file which will disable the `conversation_tag` and `company_tag` models in both the Intercom source and transform package:
```
vars:
  intercom:
    using_company_tags: False
    using_conversation_tags: False
  intercom_source:
    using_company_tags: False
    using_conversation_tags: False
```

Let me know if you still encounter an error after applying this change.

**Comment by dselman** [comment]
That didn't appear to work:

`vars` Config:

```
vars:
  mailchimp:
    using_automations: false #disable if you do not have the automation_email, automation_email, or automation_recipient_activity tables
    mailchimp_campaign_recipients: false
  intercom:
    using_company_tags: False
    using_conversation_tags: False
  intercom_source:
    using_company_tags: False
    using_conversation_tags: False
```

Error:

```
Database Error in model stg_intercom__company_tag_history_tmp (models/tmp/stg_intercom__company_tag_history_tmp.sql)
  relation "intercom.company_tag_history" does not exist
  compiled SQL at target/run/intercom_source/models/tmp/stg_intercom__company_tag_history_tmp.sql

Database Error in model stg_intercom__conversation_tag_history_tmp (models/tmp/stg_intercom__conversation_tag_history_tmp.sql)
  relation "intercom.conversation_tag_history" does not exist
  compiled SQL at target/run/intercom_source/models/tmp/stg_intercom__conversation_tag_history_tmp.sql
```

**Comment by fivetran-joemarkiewicz** [comment]
@dselman my apologies another customer encountered this same issue and we have a [pull request](https://github.com/fivetran/dbt_intercom_source/pull/6) open that fixes the disabling of these tmp tables in the source package if they are not used. 

We will try to get this pull request approved and merged soon so you can successfully run this package and use your modeled Intercom data. I will let you know once the PR is merged!

**Comment by fivetran-joemarkiewicz** [comment]
@dselman our [Intercom v0.1.2](https://hub.getdbt.com/fivetran/intercom/latest/) release is now live! This release fixes the table disable variables bug you were encountering as well as some additional updates as well.

You can now update your package dependency to install the most recent Intercom release.
```
packages:
  - package: fivetran/intercom
    version: 0.1.2
```

Let me know if you have any questions!

**Comment by dselman** [comment]
That fixed it. Thank you very much! 🚀

**Comment by fivetran-kristin** [comment]
@dselman Can you set up Fivetran to run a `dbt run --full-refresh` and see if that works?

**Comment by fivetran-joemarkiewicz** [comment]
Hi @dselman did Kristin's recommendation fix your issue? If not, let us know and we can help further. Otherwise we can close this issue.

Thanks!

**Comment by fivetran-joemarkiewicz** [comment]
Hi @dselman I assume the issue is no longer relevant and has been resolved. As such I will close this issue. If that is not the case, and the issue still persists, please open a new issue and we may work further from there.

Thank you!
